Source/DafnyRuntime/DafnyRuntimeDafny/Makefile

```diff
@@ -0,0 +1,31 @@
+
+DAFNY = dotnet run --project ../../DafnyDriver --no-build --
```

Comment:
That was my original plan, but now since the generated code is checked in we still build the whole Dafny first, runtimes included, and then just verify that this Dafny generates the same code. This line is really just a nice way to stay platform-independent so I don't have to worry about `dafny` vs. `Dafny.exe`. I'll comment.